OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -1570,9 +1823,132 @@ namespace OloEngine
+                glm::vec3 camUp = glm::normalize(glm::vec3(cameraTransform[1]));
+                glm::vec3 camPos = glm::vec3(cameraTransform[3]);
+
+                RenderParticleSystems(camRight, camUp, camPos, nearClip, farClip);
+
+                ParticleBatchRenderer::EndBatch(); });
+        }
+
         Renderer3D::EndScene();
     }
 
+    void Scene::RenderParticleSystems(const glm::vec3& camRight, const glm::vec3& camUp, const glm::vec3& camPos, f32 nearClip, f32 farClip)
```

Clang (14.0.6)

[warning] method 'RenderParticleSystems' can be made static

(readability-convert-member-functions-to-static)